src/app/search/search.component.html

```diff
@@ -167,6 +167,12 @@
             </span>
           </div>
         </div>
+
+        <div class="alert alert-info" *ngIf="hits.length > query_size - 1">
+          <p><strong>Notice: </strong>Your search has returned greater than {{ query_size - 1 }} results, however only up to {{ query_size - 1 }} results are shown.
```

Comment:
If I understand correctly, this isn't actually "up to {{ query_size - 1 }} results", it should be precisely "{{ query_size - 1 }} results" 
i.e. "Your search has returned greater than {{ query_size - 1 }} results, however only {{ query_size - 1 }} results are shown. We recommend that you narrow your search to find more relevant results."

Reply:
Yes good catch